tests/features/camel-core/src/main/resources/OSGI-INF/blueprint/route2.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <camelContext xmlns="http://camel.apache.org/schema/blueprint" id="ctx2">
+        <route id="leg-2">
+            <from uri="direct-vm:nextOne"/>
+            <log message="second leg"/>
+            <to uri="vm:test-vm"/>
+        </route>
+        <route id="leg-3">
+            <from uri="vm:test-vm"/>
```

Comment:
Shouldn't it be in another blueprint file since it is supposed to support communication across CamelContext instances?

Reply:
yes, added a third route